src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractClassNameCheck.java

```diff
@@ -28,17 +28,41 @@
 
 /**
  * <p>
- * Ensures that the names of abstract classes conforming to some
- * regular expression and check that {@code abstract} modifier exists.
+ * Ensures that the names of abstract classes conforming to some regular
+ * expression and check that {@code abstract} modifier exists.
  * </p>
  * <p>
- * Rationale: Abstract classes are convenience base class
- * implementations of interfaces, not types as such. As such
- * they should be named to indicate this. Also if names of classes
- * starts with 'Abstract' it's very convenient that they will
- * have abstract modifier.
+ * Rationale: Abstract classes are convenience base class implementations of
+ * interfaces, not types as such. As such they should be named to indicate this.
+ * Also if names of classes starts with 'Abstract' it's very convenient that
+ * they will have abstract modifier.
+ * </p>
+ * <ul>
+ * <li>
+ * Option {@code format} - Specify valid identifiers. Default value is
```

Comment:
is it better to start using term "Property" here (instead of Option) ?
Right term is property accoding to our xdocs. Option is unfortunate synonym that I used for some time, and it some how spread among users and contributors.

Reply:
If you want me to swap it, then I am fine with that.

Reply:
I'm not sure where you are seeing `wrap`.
I understood you want me to change `option` to `property`, but you formed it as a question so I wasn't sure you wanted me to make the change. I agree that property is probably better, so I will make the change.

Reply:
Done.